create-k8s-chained-sessions.sh

```diff
@@ -61,8 +76,5 @@ PARENT_SESSION_NAMES="panorama-k8s-playground panorama-k8s-playground-2 panorama
 
 for session in $PARENT_SESSION_NAMES
```

Comment:
Could you remove `panorama-k8s-integration` from the `PARENT_SESSION_NAMES` array as part of this? that env no longer exists.